### PR TITLE
Auth docs update

### PR DESCRIPTION
**Pull request by dudiiiiiiii**

### CODE REVIEW

File: docs/authentication.md [hunk 1]
```diff
@@ -63,12 +103,49 @@ sequenceDiagram
...
+    -   Every 1/3 of `accessExp` (1/3 of Backend access token), AuthService refreshes the token with Backend.
+    -   Backend returns a new access token in cookies and an updated `accessExp`.
+
+-   **Logout process**:
+    -   User initiates logout.
+    -   AuthService sends a session invalidation request to Backend.
+    -   Backend deletes session cookies, confirming logout.
+    -   AuthService clears the user's data.
+
+## Default vaules
```
Comment (grzanka) [inline docs/authentication.md:145]:
```suggestion
## Default values
```

File: docs/authentication.md [hunk 2]
```diff
@@ -33,22 +33,62 @@ sequenceDiagram
...
+
+-   **Logout process**:
+    -   User initiates logout.
+    -   AuthService logs user out of Keycloak and clears user data.
+    -   AuthService invalidates Backend session.
+    -   Backend deletes session cookies.
+
+## Our own database of users validation
 
 Overview of login and logout process while in demo or dev modes
```
Comment (grzanka) [inline docs/authentication.md:93]: BTW - do we have login process in demo mode ? I though that demo mode means login disables, like in github pages deployment,

File: docs/authentication.md [hunk 3]
```diff
@@ -33,22 +33,62 @@ sequenceDiagram
...
     User ->> AuthService: Logout
+    AuthService ->> Keycloak: Logout (GET /auth/realms/{our-realm}/protocol/openid-connect/logout)
     AuthService ->> Backend: Invalidate session (DELETE /auth/logout)
     Backend ->> AuthService: Response with cookies deleted
-    AuthService ->> Keycloak: Logout
     AuthService ->> User: Clear user data
 ```
 
-## Non-Keycloak
+-   **User requests login**: User initiates login with SSO.
```
Comment (grzanka) [inline docs/authentication.md:55]: You could use numbered list (best with manually hardcoded numbers), then those numbers could correspond to numbers of items on the diagram

File: docs/authentication.md (hunk 2 above)
Comment (dudiiiiiiii) [reply docs/authentication.md:93]: i meant dev deployment with docker. will fix

File: docs/authentication.md [hunk 4]
```diff
@@ -63,12 +159,104 @@ sequenceDiagram
...
+
+1️2. **Clear user data**
+
+-   AuthService clears the user's data from the local session.
+
+---
+
+## **Default Token Values**
+
+**Keycloak token values** (defined by Keycloak server on Ares):
```
Comment (grzanka) [inline docs/authentication.md:252]:
```suggestion
**Keycloak token values** (defined by Keycloak server in plgrid infrastructure `sso.plgrid.pl`):
```

**Review by grzanka — APPROVED** [review]

**Pull request merged**